### PR TITLE
[A11y] Improve focus management + navigation in widget ajax

### PR DESCRIPTION
**Pull request by wiffbi**
This PRs improves focus management in the widget when using any navigation methods other than opening the iframe-overlay – this usually only happens on event-series or when the widget is included on a organizer-level. To make screen-readers recognize new content loaded dynamically and not lose focus-order, focus is moved to the widgets container and the screen-reader reads the aria-label, which is the event’s name (empty for organizer level, could be improved if a subevent is selected).

Making the whole container aria-live is not helping here, as aria-live will read the whole container once it is loaded.

Usually only interactive elements are focusable (depending on browser). To make the container focusable (at least in Safari with VoicerOver), we need to add a `tabindex` to it. This makes it a focusable object, which ideally should have an aria-label. I initially thought of only adding the aria-label and the tabindex once the widget is navigated inside, as the organizer-view does not necessarily provide any useful label for the whole widget. The widget-container still needs to be focusable though (e.g. when navigating back from an event to the organizer overview), so this PR adds tabindex with a potentially empty aria-label (only empty in organizer-level widget). Not ideal, but IMHO the best option.

Furthermore for best experience when navigating inside the widget’s calendar, it adds an optional focus-parameter to reload to focus the current calendar element instead of the whole widget. It also adds some aria-labels to improve understanding when navigating the calendar.

All navigation links are made actual links (rel=button does not make a link with no href a tab-target with Safari/VoiceOver). Also in the best interest of a11y, all calendar navigation links are now the correct links to the actual pretix-shop so there is a static-html-page fallback instead of loading content via ajax. So e.g. an open-link-in-new-window/tab now works as expected.

### CODE REVIEW

File: src/pretix/static/pretixpresale/js/widget/widget.js [hunk 1]
```diff
@@ -1097,6 +1104,9 @@ Vue.component('pretix-widget-event-form', {
         this.$root.$off('focus_voucher_field', this.focus_voucher_field)
     },
     computed: {
+        href: function () {
+            return this.$root.event?.event_url || this.$root.parent_stack[this.$root.parent_stack.length-1];
```
[the remark this answers is not in the document]
Comment (raphaelm) [reply src/pretix/static/pretixpresale/js/widget/widget.js:1108]: > How do you mean confusing? As in „it presents itself as a link, but it does not work as a normal link“ (ie triggers a full page-reload)? IMHO that is often how web-apps work – especially when they handle window.history properly. Or confusing that it is an external URL?

With the type of webhook you are refering to, links do not work as normal links, but the URL in the href attribute is still an URL that would bring me to the same place if it were a normal link. Here, it points to an URL that I will not visit if I stay in the widget. In other words: If I hover over the link, I'll see an external domain, so I'd expect this click to go somewhere else but then it doesn't. This might even prevent someone who does not want to leave the page from clicking the link in an extreme case? But maybe I'm overthinking this any nobody will care :

[the remark this answers is not in the document]
Comment (wiffbi) [reply src/pretix/static/pretixpresale/js/widget/widget.js]: As discussed I removed all href
